### DISCUSSION TITLE
FieldError in NetBox Branching Plugin on NetBox docker

### DISCUSSION THREAD

**Discussion by BrynjarFAune** [netbox-community/netbox#18111] [Help Wanted!]
I’m encountering an issue with the netbox-branching plugin after integrating it into my NetBox Docker image. 
The plugin appears in the GUI, and I can create new branches, but I cannot switch between branches, and accessing the branches plugin page results in a server error with the message below.

```plaintext
<class 'django.core.exceptions.FieldError'>

Cannot resolve keyword 'is_active' into field. Choices are: applied_changes, bookmarks, changediff, comments, conflicts, created, custom_field_data, description, events, id, jobs, journal_entries, last_sync, last_updated, merged_by, merged_by_id, merged_time, name, owner, owner_id, schema_id, status, subscriptions, tagged_items, tags
```

### Steps Taken:

- Created a custom docker image according to [these instructions](https://github.com/netbox-community/netbox-docker/wiki/Using-Netbox-Plugins)
- Added the plugin to plugins.py and plugin_requirements.txt
- created local_settings.py.
- Granted CREATE permissions to the netbox user on the database, following [these instructions](https://github.com/netboxlabs/netbox-branching)
- Ran migrations and rebuilt/restarted the Docker container.

### Environment:

- Python version: 3.12.3
- NetBox version: 4.1.6
- netbox_branching: 0.5.2
- PostgreSQL 16.4

### Question:

How can I resolve the field error?

**Comment by candlerb** [comment]
> How can I resolve the field error?

Raise the issue with the plugin author, via its [github page](https://github.com/netboxlabs/netbox-branching/issues)
